[TW-671] Websocket refactor (#5204)
postmanlabs/postman-docs: src/pages/docs/sending-requests/websocket/listen-to-socketio-events.md

---
title: "Listen to Socket.IO events"
updated: 2023-11-01
contextual_links:
  - type: section
    name: "Additional resources"
  - type: subtitle
    name: "Videos"
  - type: link
    name: "WebSocket Requests | Postman Level Up"
    url: "https://youtu.be/H-7EZVj9D-k"
  - type: link
    name: "Set Up a WebSockets Server in Node.js"
    url: "https://youtu.be/e__oWJ4wNtw"
  - type: subtitle
    name: "Blog posts"
  - type: link
    name: "Save and Document WebSocket Requests in Collections"
    url: "https://blog.postman.com/save-and-document-websocket-requests-in-collections/"
  - type: link
    name: "Powering home automation with WebSocket APIs"
    url: "https://blog.postman.com/powering-home-automation-with-websocket-apis/"
  - type: link
    name: "Postman Supports Socket.IO"
    url: "https://blog.postman.com/postman-now-supports-socket-io/"
---

> In Socket.IO, you have the ability to listen to specific events. The **Response** pane will display the received events for which you've added listeners. Events will be color-coded by event to make them easier to find.

Reviewer: ```suggestion
In Socket.IO, you have the ability to listen to specific events. The **Response** pane displays the received events for which you’ve added listeners. Events are color coded by event to make them easier to find.
```

To listen to a new event, do the following:

1. Select the **Events** tab.
1. Select **+** or **Add event**.
1. Enter a name and description for the event.
1. Select the **Listen** switch next to an event to turn listening on or off for that event.

>     > Hover over the **Listen** switch and select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icons/icon-delete-red-v10-19.jpg#icon" width="14px"> to delete the event listener.

Reviewer: ```suggestion
    > To delete the event listener, hover over the **Listen** switch and select the delete icon <img alt="Delete icon" src="https://assets.postman.com/postman-docs/icons/icon-delete-red-v10-19.jpg#icon" width="14px">.
```
